### ISSUE TITLE
notused

### ISSUE THREAD

**Issue by setwindowhook** [SMorettini/CNNs-on-CHB-MIT#7]
https://github.com/SMorettini/CNNs-on-CHB-MIT/blob/b85441c8964b30cd9664eec3044fc792031c0872/DataserToSpectogram.py#L143

why,there

return (data.shape[1]-t*_SIZE_WINDOW_SPECTOGRAM)*-1

not     

return (data.shape[1]-(t * movement+_SIZE_WINDOW_SPECTOGRAM))*-1

**Comment by SMorettini** [comment]
You are right, with my code a window of data interictal is considered 2 times.

I would like to update the code with your suggestion but before I want to be sure that is not causing the error. Do you have the possibility to make the change and run the code and let me know if everything works fine with your suggestion? 

@setwindowhook I asked you because I don't have anymore the environment set up for running the code.
